### PR TITLE
Remove govet test

### PR DESCRIPTION
**Pull request by elfosardo**
It's included in golint in PR https://github.com/metal3-io/baremetal-operator/pull/825

### CODE REVIEW

**Comment by elfosardo** [comment]
after the 2 PRs merge, we can remove also the scripts from hack in baremetal-operator

File: prow/config/config.yaml
Comment (fmuyassarov) [inline prow/config/config.yaml:423]: This is CAPM3 repo, you need to remove govet from https://github.com/metal3-io/project-infra/blob/ebe2d90c26bb3bf284fd34c2c171a82058249da6/prow/config/config.yaml#L252-L265.

**Review by fmuyassarov — CHANGES_REQUESTED** [review]

**Review by fmuyassarov — COMMENTED** [review]
/hold 
until https://github.com/metal3-io/baremetal-operator/pull/825 gets merged.

**Comment by fmuyassarov** [comment]
> /approve

Seems I don't rights here. 
@maelk Would you mind to approve/lgtm this if looks okay?

**Review by maelk — APPROVED** [review]

**Comment by elfosardo** [comment]
https://github.com/metal3-io/baremetal-operator/pull/825 merged, this should be good to go

**Review by fmuyassarov — COMMENTED** [review]
/lgtm

**Pull request merged**
